libs/ui-toolkit/src/components/button/button.tsx

```diff
@@ -139,7 +152,8 @@ const getClasses = (
 
 const classes = (
```

Comment:
suggestion (non-blocking): 

1. I'd prefer to just type these as as string, Variant, and boolean. That way when you are looking at this function you know what args it takes and you don't need to check the props to see
2. I'm not sure why we need this function _and_ `getClasses` can we just make it a single get classes function?

Reply:
1. Sure. 2. Yep!